src/test/java/org/broadinstitute/hellbender/engine/CRAMReferenceIntegrationTest.java

```diff
@@ -70,7 +70,7 @@ public void testWrongRef(String fileIn, String extOut, String referenceFile) thr
                 "-R", reference.getAbsolutePath()
         };
         runCommandLine(args);
-    }*/
+    }
 
     @DataProvider(name="testingDataWrongRef")
     public Object[][] testingDataWrongRef() {
```

Comment:
Name cramtest.fasta to make it clear that it's the wrong reference for cramtest.cram